components/builder-originsrv/src/migrations/origin_packages.rs

```diff
@@ -267,5 +267,35 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                         RETURN;
                     END
                     $$ LANGUAGE plpgsql STABLE"#)?;
+    migrator.migrate("originsrv",
```

Comment:
Yeah, it will, because the migrator now hashes the SQL content to be migrated as uses that to determine whether or not to run a migration.  We no longer have to worry about changing the initial string passed to the `migrate` function.  That's more just for categorizing the migrations at this point.